kobo/apps/accounts/templatetags/customize_email_content.py

```diff
@@ -0,0 +1,25 @@
+from django import template
+from django.utils.translation import gettext_lazy as t
+
+from kobo.apps.accounts.models import EmailContent
+
+register = template.Library()
+
+
+@register.filter
+def email_template(email_name):
+    email_contents = EmailContent.objects.values(
+        'section_name', 'content'
+    ).filter(email_name=email_name)
+    email_content_dict = {}
+    for email_content in email_contents:
+        email_content_dict[email_content['section_name']] = email_content['content']
+    return email_content_dict
+
+
+@register.simple_tag
+def get_variables(section_content, activate_url, user):
```

Comment:
The name sounds generic, but is really activation link specific. Let's rename it unless there is a strong feeling it should be made more general purpose.

Reply:
It's intended to be generic because it covers the activation link, the username, and anything else we may be asked to create placeholders for. If we do one tag for each one, these templates will get extremely messing quick. I can make it slightly less generic as `replace_placeholders` or something